pg_search/src/postgres/options.rs

```diff
@@ -218,15 +231,20 @@ pub unsafe extern "C-unwind" fn amoptions(
             offset: offset_of!(BM25IndexOptionsData, key_field_offset) as i32,
         },
         pg_sys::relopt_parse_elt {
-            optname: "layer_sizes".as_pg_cstr(),
+            optname: "foreground_layer_sizes".as_pg_cstr(),
             opttype: pg_sys::relopt_type::RELOPT_TYPE_STRING,
-            offset: offset_of!(BM25IndexOptionsData, layer_sizes_offset) as i32,
+            offset: offset_of!(BM25IndexOptionsData, foreground_layer_sizes_offset) as i32,
```

Comment:
I think we should keep this named as `layer_sizes`, otherwise it'll be a breaking change for existing users as their current DDL would no longer be compatible.